backend/routes/post.js

```diff
@@ -92,30 +134,98 @@ router.post('/add-user', async (req, res) => {
     });
 
 
-    router.get('/popular-playlists', (req, res) => {
-      User.aggregate([
-        { $unwind: '$favorite' },
-        {
-          $group: {
-            _id: '$favorite',
-            count: { $sum: 1 }
-          }
-        },
-        { $match: { count: { $gt: 1 } } },
-        { $sort: { count: -1 } },
-        { $limit: 10 }
-      ])
-        .then(result => {
-          // Map the playlist IDs from the result array
-          const popularPlaylists = result.map(item => item._id);
-          res.json(popularPlaylists);
-        })
-        .catch(error => {
-          console.error(error);
-          res.status(500).json({ error: 'Internal server error' });
-        });
-    });
-    
    
+      
+    // router.put('/save-notes/:id', async (req, res) => {
```

Comment:
Remove all the commented code from GitHub repo, later if you need you can refer previous commits.